### ISSUE TITLE
Error on WebTestMixin initialisation if `DEFAULT_AUTHENTICATION_CLASSES` is missing

### ISSUE THREAD

**Issue by hugorodgerbrown** [django-webtest/django-webtest#99]
I recently upgraded from 1.9.4 to 1.9.5, and our tests are now failing:

```
def _setup_auth_class(self):
    class_name = 'django_webtest.rest_framework.WebtestAuthentication'
>   auth_classes = settings.REST_FRAMEWORK['DEFAULT_AUTHENTICATION_CLASSES']

E   KeyError: 'DEFAULT_AUTHENTICATION_CLASSES'
```

It looks like the issue is from this PR (https://github.com/django-webtest/django-webtest/pull/92), where `_setup_auth_class` assumes that the key `DEFAULT_AUTHENTICATION_CLASSES` exists in the `REST_FRAMEWORK` settings dictionary. This dict is set if `REST_FRAMEWORK` doesn't exist in the settings (see https://github.com/django-webtest/django-webtest/blob/master/django_webtest/__init__.py#L236-L237), but if the settings **do** exist, but do **not** contain `DEFAULT_AUTHENTICATION_CLASSES`, then this error will occur.

Our settings:

```python
REST_FRAMEWORK = {
    # Any authenticated user can access the API (overridable per-view)
    "DEFAULT_PERMISSION_CLASSES": ["rest_framework.permissions.IsAuthenticated"],
    "DATETIME_FORMAT": "%Y-%m-%d %H:%M:%S%z",
    "JSON_UNDERSCOREIZE": {"no_underscore_before_number": True},
    "DEFAULT_RENDERER_CLASSES": (
        "djangorestframework_camel_case.render.CamelCaseJSONRenderer",
    ),
    "DEFAULT_PARSER_CLASSES": (
        "djangorestframework_camel_case.parser.CamelCaseJSONParser",
        "rest_framework.parsers.JSONParser",
        "rest_framework.parsers.FormParser",
        "rest_framework.parsers.MultiPartParser",
        "rest_framework.parsers.FileUploadParser",
    ),
}
```

**Comment by hugorodgerbrown** [comment]
NB editing test settings to add `DEFAULT_AUTHENTICATION_CLASSES` also works as a hack.

**Comment by bdauvergne** [comment]
It's fixed by #101 and release 1.9.6 I think.
